File: templates/python/fr/text/code/99beepers.py
# Fichier: PlaceMany Jetons .py
# -----------------------------
# Places 42 jetons utilisant un boucle for
from karel.stanfordkarel import *

def main():
   avancer()
   # répéter laisser_jeton plusieurs fois
   for i in range(42):
      laisser_jeton()
   avancer()

File: templates/java/fr/text/code/99beepers.java
/*
 * Fichier: PlaceBeaucoupJetons.java
 * --------------------------------
 * Places 42 jetons utilisant un boucle for
 */
import stanford.karel.*;
public class PlaceBeaucoupJetons extends Karel {
   public void run() {
      avancer();
      /* répéter laisserJeton plusieurs fois */
      for(int i = 0; i < 42; i++) {
      	laisserJeton();
      }
      avancer();
   } 
}